pyacq/core/stream.py

```diff
@@ -154,33 +262,55 @@ def __init__(self, **kargs):
         #send or cpy to buffer
         if self.params['transfermode'] == 'plaindata':
```

Comment:
Maybe for implementation.
But I like the idea to deal with StreamSender or StreamReceiver inside nodes without worring wich transfer mode they are, so one class.